bin/render-build.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+# exit on error
+set -o errexit
+
+bundle install
+bundle exec rake db:truncate_all DISABLE_DATABASE_ENVIRONMENT_CHECK=1
+bundle exec rake db:migrate
+bundle exec rake db:seed
```

Comment:
A suggestion for next upcoming projects: 
I suggest you deploy your project on a different branch than the default.
So you can keep up your deployment branch separately from the branch you are constantly working on and making changes.

Reply:
@wandergithub I did not understand very well, but I tell you that in render.com it is very easy and fast to change the branch used to deploy the app